lib/coxir/struct/channel.ex

```diff
@@ -160,6 +162,146 @@ defmodule Coxir.Struct.Channel do
     API.request(:post, "channels/#{channel}/messages/bulk-delete", %{messages: messages})
   end
 
```

Comment:
Can we move the `edit/2` function up here?
This applies to all the modified modules.

Reply:
Sure,